account/src/main/java/com/user/account/entity/Transactions.java

```diff
@@ -0,0 +1,35 @@
+package com.user.account.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table (name ="transactions" )
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter 
+@Setter
+public class Transactions {
+	
+	@Id
+	private long transactionID;

```

Comment:
why they are not `private final`?

Reply:
If I declare a variable as private final then i need to initialize it at the time of declaration or in the constructor. Since it is my id I can not initialize it at the time of declaration. And using Lombok for constructure, how to handle the error?

Reply:
Don't keep ID as final, make others final then.

Reply:
I need noArgConstructor for my implementation. If my variable will be a final variable, then i will not able to use noArgConstructor.

Reply:
Why'd you need `noArgConstructor `?

Reply:
noArgConstructor is needed for data ingestion into the table